### ISSUE TITLE
sorting by score spotlight

### ISSUE THREAD

**Issue by petersopko** [kodadot/nft-gallery#3001]
> * [ ]  sorting by `Score` doesn't seem to work?

> Yes, it is the same as online env (https://kodadot.xyz/spotlight)

> After some tries,  I found that it needs to extend some function at `rubick` if it supports sorting by 'score'.
> Maybe need to create a new issue to make it.

https://github.com/kodadot/nft-gallery/issues/2991

- add sorting by creator as well

**Comment by petersopko** [comment]
@Jarsen136 I've followed up here from #2991

**Comment by zhengow** [comment]
👋

**Comment by kodabot** [comment]
ASSIGNED - @zhengow 🔒 LOCKED -> Monday, July 4th 2022, 19:06:44 UTC -> 36 hours

**Comment by zhengow** [comment]
Would pls review 
[rubick#107](https://github.com/kodadot/rubick/pull/107)
which is for this issue

**Comment by kodabot** [comment]
ASSIGNMENT EXPIRED - @zhengow has been unassigned.

**Comment by zhengow** [comment]
> @petersopko Would pls review [rubick#107](https://github.com/kodadot/rubick/pull/107) which is for this issue

Any progress on this 👀

**Comment by petersopko** [comment]
> Any progress on this 👀

well this one needs to be done first:
- https://github.com/kodadot/rubick/pull/107
